docs/data/date-pickers/custom-opening-button/CustomOpeningIcon.js

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { createSvgIcon } from '@mui/material/utils';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DemoContainer } from '@mui/x-date-pickers/internals/demo';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+
+const FlightTakeoffIcon = createSvgIcon(
+  <path d="M2.5 19h19v2h-19v-2zm19.57-9.36c-.21-.8-1.04-1.28-1.84-1.06L14.92 10l-6.9-6.43-1.93.51 4.14 7.17-4.97 1.33-1.97-1.54-1.45.39 2.59 4.49L21 11.49c.81-.23 1.28-1.05 1.07-1.85z" />,
+  'FlightTakeoff',
+);
+
+function MuiIcon() {
```

Comment:
If you have a better SVG to use.
I did not want to store a new SVG in the `public` folder just for this demo, but if you think it's best I can add one there.

Reply:
I think that it makes perfect sense if we want to showcase the fact that you can use custom SVG paths or even images as components in this demo. 👌 
And the given icon nicely aligns with the provided `label`. 👍